soc/nordic/nrf54l/soc.h

```diff
@@ -13,7 +13,7 @@
 
 #include <soc_nrf_common.h>
 
-#define FLASH_PAGE_ERASE_MAX_TIME_US 8000UL
+#define FLASH_PAGE_ERASE_MAX_TIME_US 42000UL
```

Comment:
This value is not used by the driver.
FLASH_SLOT_WRITE is used instead.

Reply:
it used here: https://github.com/zephyrproject-rtos/zephyr/blob/e1c51f9a7228da4bfd8ad53acf89fb8c736dc705/drivers/flash/soc_flash_nrf.h#L35

isnt it?